clients/python-apiserver-client/python_apiserver_client/params/cluster.py

```diff
@@ -0,0 +1,193 @@
+from .headnode import *
+from .workernode import *
+
+class Environment(enum.Enum):
+    DEV = 0
+    TESTING = 1
+    STAGING = 2
+    PRODUCTION = 3
+
+
+class ClusterSpec:
+    """
+    ClusterSpec is used to define Ray cluster.
+    It provides APIs to create, stringify, convert to dict and json.
+
+    Methods:
+    - Create cluster spec from: gets the following parameters:
```

Comment:
Wording? "from: gets"?

Reply:
Also, I don't understand the "Methods:" line. This looks like the constructor doc comment.